src/schema/v2/homeView/index.ts

```diff
@@ -34,13 +40,35 @@ const SectionConnection: GraphQLFieldConfig<any, ResolverContext> = {
   },
 }
 
+const Section: GraphQLFieldConfig<void, ResolverContext> = {
+  type: HomeViewSectionType,
+  description: "A home view section",
+  args: {
+    id: {
+      description: "The ID of the section",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+  },
+  resolve: (_root, { id }, context) => {
+    const { meLoader } = context.authenticatedLoaders
+
+    if (!meLoader) throw new Error("You must be signed in to see this content.")
+
+    if (id.length === 0) {
```

Comment:
```suggestion
    if (!id) {
```

Reply:
The argument is already enforced by the schema to be non-null. I think this check (which I borrowed from a similar resolver) guards against an empty string.